components/main/GlobalAlert.tsx

```diff
@@ -69,24 +91,61 @@ function BookmarkAlert() {
   );
 }
 
-function CojAlert() {
-  const pathName = useLocation().pathname;
+type OutstandingActionsAlertProps = {
+  unsignedCoJ: boolean;
+  inProgressFormR: boolean;
+  importantInfo: boolean;
+};
+
+function ActionsSummaryAlert({
+  unsignedCoJ,
+  inProgressFormR,
+  importantInfo
+}: Readonly<OutstandingActionsAlertProps>) {
+  const ACTION_LINK = (
+    <span>
+      Please click <Link to="/">here</Link> for details.
+    </span>
+  );
+  const importantInfoText =
+    "You have important information about your Form R submission dates you might want to check.";

```

Comment:
Maybe something a bit less wordy (and polite, ugh :>), like 'Please review your Form R submissions'?